go.mod

```diff
@@ -85,6 +85,7 @@ require (
 )
 
 replace (
+	github.com/chaos-mesh/api => ./api/v1alpha1
```

Comment:
It seems that there is no need to extract it out for us now. (The kubernetes/api was extracted to avoid diamond dependency problem).

We could add tag to release the [submodule](https://github.com/go-modules-by-example/index/tree/master/009_submodules) later.